packages/react/src/patterns/sections/CardSectionImages/README.md

```diff
@@ -0,0 +1,141 @@
+    },
+  },
+  {
+    image: {
+      defaultImage: 'https://picsum.photos/id/1018/1056/480',
+      alt: 'cards with image',
+    },
+    eyebrow: 'Blog',
+    heading: 'Natural language processing.',
+    cta: {
+      href: 'https://www.example.com',
+    },
+  },
+  {
+    image: {
+      defaultImage: 'https://picsum.photos/id/1076/1056/480',
+      alt: 'cards with image',
+    },
+    eyebrow: 'Topic',
+    heading: 'Natural language processing.',
+    cta: {
+      href: 'https://www.example.com',
+    },
+  },
+  {
+    image: {
+      defaultImage: 'https://picsum.photos/id/102/1056/480',
+      alt: 'cards with image',
+    },
+    eyebrow: 'Blog',
+    heading: 'Serving society ethically in the age of Artificial Intelligence.',
+    cta: {
+      href: 'https://www.example.com',
+    },
+  },
+  {
+    image: {
+      defaultImage: 'https://picsum.photos/id/1032/1056/480',
+      alt: 'cards with image',
+    },
+    eyebrow: 'Topic',
+    heading: 'Serving society ethically in the age of Artificial Intelligence.',
+    cta: {
+      href: 'https://www.example.com',
+    },
+  },
+];
+
+function App() {
+  return (
+    <CardSectionImages cards={CardSectionImages} heading="Read more about it" />
+  );
+}
+
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+> 💡 Don't forget to import the card-section styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+## Props
+
+| Name      | Required | Data Type | Default Value | Description                                                                                                                                                       |
+| --------- | -------- | --------- | ------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `heading` | YES      | String    | null          | Cards with or without images heading.                                                                                                                             |
+| `cards`   | YES      | Array     | null          | Array of card objects. See [CardSection](https://github.com/mkothur/ibm-dotcom-library/blob/master/packages/react/src/patterns/sections/CardSection/README.md) 👀 |
+| `theme`   | NO       | String    | 'white'       | Color theme for pattern. See `theme` below.                                                                                                                       |
+
+## Card Props
+
+Card accepts the folloing information for this `CardSectionImages` pattern
+
+> 👀 See
+> [Card](https://github.com/mkothur/ibm-dotcom-library/blob/master/packages/react/src/patterns/sub-patterns/Card/README.md)
+> and 👀
+> [Card Object](https://github.com/mkothur/ibm-dotcom-library/blob/master/packages/react/src/patterns/sections/CardSection/__stories__/data/cards.json)
```

Comment:
This should go higher up like below the getting started. It's not related specifically to this section.

Reply:
Okay done